.travis.yml

```diff
@@ -13,10 +16,11 @@ android:
   components:
   - tools
   - platform-tools
+  - tools
```

Comment:
Without that it didn't update sdk, then i found it out after some research, but i don't remember actually it's been a while :D